src/model-viewer-base.ts

```diff
@@ -325,6 +326,22 @@ export default class ModelViewerElementBase extends UpdatingElement {
     return this[$canvas].toDataURL(type, encoderOptions);
   }
 
+  async toBlob(mimeType?: string, qualityArgument?: number): Promise<Blob> {
+    return new Promise((resolve, reject) => {
+      if (!this[$canvas].toBlob) {
+         return resolve(dataURLToBlob(this[$canvas].toDataURL(mimeType)));
+      }
+
+      this[$canvas].toBlob((blob) => {
```

Comment:
[Allegedly IE11 has `HTMLCanvasElement.prototype.msToBlob`](https://caniuse.com/#feat=mdn-api_htmlcanvaselement_toblob) - could you try incorporating that?

Reply:
I tried checking for `msToBlob` within the `if (!this[$canvas].toBlob)` statement but it results in a Typescript error:

```
Property 'msToBlob' does not exist on type 'HTMLCanvasElement'.
```

Is this safe to ignore?

Reply:
The easiest thing to do would be to cast it away, kind of like what we do here: https://github.com/GoogleWebComponents/model-viewer/blob/6f4875528cee6db308bfc20037802e58821c1419/src/constants.ts#L78-L79